provisioning/baremetal_config.go

```diff
@@ -49,6 +49,9 @@ var (
 )
 
 func getDHCPRange(config *metal3iov1alpha1.ProvisioningSpec) *string {
+	if config.ProvisioningNetwork == metal3iov1alpha1.ProvisioningNetworkDisabled {
```

Comment:
I think it should be != Managed because we also don't start DHCP in the Unmanaged case

Reply:
Done